fallback.html

```diff
@@ -18,10 +18,10 @@
                   <b>
                     <a href="{{ provider.url }}">{{ provider.title }}</a> [ {{ provider.domains | join: "; "  }} ]<br>
                   </b>
-                  [ {{ provider.status.state }} on {{ provider.status.date }} ]<br>
+                  [ {{ provider.status.status }} on {{ provider.status.last_checked }} ]<br>
```

Comment:
This probably should be `provider.status` and `provider.last_checked`?

Same goes for some lines in `index.html`